### PR TITLE
Book model tests

### PR DESCRIPTION
**Pull request by plapicola**
Adds testing for core book and author relational database structure
Deploys simplecov for tracking text coverage

### CODE REVIEW

File: spec/models/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  describe 'validations' do
+    it {should validate_presence_of :title}
+    it {should validate_presence_of :year}
+    it {should validate_presence_of :page_count}
+  end
+
+  describe 'relationships' do
+    it {should have_many :bookauthors}
+    it {should have_many :authors}
+    xit {should have_many :reviews}
```
Comment (juliamarco) [inline spec/models/book_spec.rb]: Is this test supposed to be skipped?

File: spec/models/book_author_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
```
Comment (juliamarco) [inline spec/models/book_author_spec.rb]: Let's make sure that this is the right naming convention when we have our check-ins tomorrow

File: spec/models/book_spec.rb (hunk 1 above)
Comment (plapicola) [reply spec/models/book_spec.rb]: I skipped it for now as we are not beginning work on review functionality.

File: spec/models/book_author_spec.rb (hunk 2 above)
Comment (plapicola) [reply spec/models/book_author_spec.rb]: Let's open an issue to track it so we remember to ask about it on Thursday.

Comment (plapicola) [reply spec/models/book_author_spec.rb]: ...or tomorrow.

**Pull request merged**